tools/seed/main.go

```diff
@@ -444,7 +463,7 @@ func generateCodesAndStats(ctx context.Context, db *database.Database, realm *da
 					IssuingExternalID: issuingExternalID,
 				}
 				if isUserReport {
-					phoneNumber++
+					phoneNumber := phoneNumberGen.next()
```

Comment:
I don't know why this didn't see contention before, but we need to sync phone number issue...